src/main/webapp/portal/main/partials/footer.html

```diff
@@ -2,7 +2,7 @@
 	<div class="portal-power">
 		<div id="portalPageFooterLinks">
 			<div>
-				<span>© 2014, Board of Regents of the <a href="http://www.wisconsin.edu/" target="_blank">University of Wisconsin System</a></span>
+				<span>© <p style="display:inline" ng-controller="FooterController">{{date | date:'yyyy'}}</p>, Board of Regents of the <a href="http://www.wisconsin.edu/" target="_blank">University of Wisconsin System</a></span>
```

Comment:
Please move the footer controller delaratino to the top `<div>` in this partial.